.changeset/gold-mangos-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
+---
+
+Update create app template with new backend
```

Comment:
Thinking we could perhaps put a bit more info here, in particular a link to the migration docs